From 0a7da88b32015e1c7bfa82c9a24a365795e03fa3 Mon Sep 17 00:00:00 2001
From: Keiya Nobuta <h109084@gmail.com>
Date: Mon, 23 Aug 2021 00:01:07 +0900
Subject: [PATCH 4/4] include linux/of.h

Without this header, we get the following error:

symspi.c:3607:21: error: implicit declaration of function 'of_match_ptr' [-Werror=implicit-function-declaration]
|  3607 |   .of_match_table = of_match_ptr(symspi1_dt_ids_match),
|       |                     ^~~~~~~~~~~~

Signed-off-by: Keiya Nobuta <h109084@gmail.com>
---
 linux-symspi/src/symspi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/linux-symspi/src/symspi.c b/src/symspi.c
index 26366e4..88ab09a 100644
--- a/linux-symspi/src/symspi.c
+++ b/linux-symspi/src/symspi.c
@@ -28,6 +28,7 @@
 #include <linux/printk.h>
 #include <linux/spi/spi.h>
 #include <linux/gpio.h>
+#include <linux/of.h>
 #include <linux/platform_device.h>
 #include <linux/slab.h>
 #include <linux/interrupt.h>
-- 
2.25.1

